lms/templates/discussion/_thread_list_template.html

```diff
@@ -1,17 +1,6 @@
 <%page expression_filter="h"/>
 <%! from django.utils.translation import ugettext as _ %>
 <script type="text/template" id="thread-list-template">
-    <div class="forum-nav-header">
-        <button type="button" class="forum-nav-browse" id="forum-nav-browse" aria-haspopup="true">
```

Comment:
Hmmm, I thought it was a breadcrumb. IMO it is very odd that it is now a toggle, a kind-of-popup-menu, and a kind-of-breadcrumb. When we have a full breadcrumb implementation it isn't going to act like this, right? 

For the record, I don't like the way this works now, but if it is only temporary I can live with it. Eventually we should be using the breadcrumb pattern and this should not be behaving like a popup.